destination.go

```diff
@@ -13,12 +13,24 @@ func addrToPath(s string) string {
 	return strings.Replace(strings.Replace(s, ".", "_", -1), ":", "_", -1)
 }
 
+func addrInstanceSplit(addr string) (string, string) {
+	var instance string
+	// The address may be specified as server:port or server:port:instance.
+	addrComponents := strings.Split(addr, ":")
+	if len(addrComponents) == 3 {
+		instance = addrComponents[2]
+	}
+	addr = strings.Join(addrComponents[0:2], ":")
```

Comment:
could this not result in a slice out of bounds error? if original input had no ':'